Readme.md

```diff
@@ -1 +1,62 @@
-## Webchat examples with different frameworks
\ No newline at end of file
+# Helpshift web chat examples
+
+This document serves as a guide to understand the purpose of this repository, the frameworks covered, how to navigate the repository and how to contribute
+
+## Purpose
+
+This repository contains the integration of web chat using different frameworks. The goal is to provide you some sample integrations and how to avoid bad integration of web chat in your app.
```

Comment:
> ... The goal is to provide you some sample integrations and how to avoid bad integration of web chat in your app.

The goal is to provide you sample integrations to help you adhere to best practices while integrating Web Chat into your website or web app.